front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/Itinerary.tsx

```diff
@@ -110,34 +110,35 @@ function Itinerary({ path }: ItineraryProps) {
         </div>
       )}
       {origin && destination && (
-        <div className="d-flex flex-row flex-wrap">
+
+        <div className="d-flex flex-column flex-sm-row flex-wrap"style={{ gap: '0.5rem' }}>
```

Comment:
You should put the style in the css file or find the associated class in bootstrap if it exists